src/main/java/de/jetwick/snacktory/JResult.java

```diff
@@ -128,8 +129,17 @@ public String getAuthorName() {
     public JResult setAuthorName(String authorName) {
         this.authorName = authorName;
         return this;
-    }	
-	
+    }
+
+    public String getRawAuthorName() {
```

Comment:
Added test cases for rawAuthorName https://github.com/skyshard/snacktory/pull/47/files#diff-54669b9ba2e67fc2468e33046ee1e1b8R3534